raiden/network/pathfinding.py

```diff
@@ -57,12 +89,36 @@ def get_random_service(
     return url, address
 
 
+@dataclass
+class PFSConfiguration:
```

Comment:
I don't agree with what @rakanalh wrote [here](https://github.com/raiden-network/raiden/pull/3734/files/f984dda1368bd341fa648682dd892c2cfc3a714d#discussion_r273870560).

Not every case where NamedTuple is used should be changed by a dataclass. Dataclasses are by default mutable, while NamedTuples are immutable. Since this is just used as a return value I would just keep it as a named tuple. If a bit more logic is added to the data class that's when i would use a dataclass.

But at the very least, if you really wanted to go with a dataclass even for this I would use `frozen=true` to make it immutable. But keep in mind dataclasses are a tad bit less performant than NamedTuples.